Package.swift

```diff
@@ -54,6 +65,19 @@ let package = Package(
       name: "TokamakShim",
       dependencies: [.target(name: "TokamakDOM", condition: .when(platforms: [.wasi]))]
     ),
+    .target(
+      name: "TokamakStatic",
+      dependencies: [
+        "TokamakCore",
+        "TokamakDOM",
+      ]
```

Comment:
I wonder if it could work the other way around where `TokamakDOM` depends on this new module? I also thought that `TokamakDOM` can't be built for platforms other than WASI, at least from the naming perspective DOM is only available in browser environments which currently embed WASI through the entrypoint files provided by `carton`.

Reply:
Yes, I agree that it would be nice to go the other way. I'll spend more time on it to swap those.

Reply:
Maybe some sort of `TokamakHTML` pseudo-renderer with DOM and static adaptors that produce the actual output?